core/templates/base-components/base-content.component.ts

```diff
@@ -53,8 +53,11 @@ export class BaseContentComponent {
   ) {}
 
   ngOnInit(): void {
-    if (this.windowRef.nativeWindow.location.hostname ===
-      'oppiaserver.appspot.com') {
+    /**
+     * Redirect occasional developer using the old appsport url to
```

Comment:
Please proofread your comments. For example, appspot is misspelled.

Also, it's weird to say "occasional developer" in this context. I suggest: "Redirect any developers using the old appspot URL to the test server (see issue #7867 for details)."

Reply:
done.